app/src/main/java/org/android/androidsubeen/ThreadActivity.kt

```diff
@@ -0,0 +1,112 @@
+package org.android.androidsubeen
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.*
+import androidx.appcompat.app.AppCompatActivity
+import org.android.androidsubeen.databinding.ActivityThreadBinding
+import java.io.IOException
+import java.io.InputStream
+import java.net.HttpURLConnection
+import java.net.URL
+
+class ThreadActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityThreadBinding
+    val myHandler = MyHandler()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityThreadBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        clickEvent()
+    }
+
+    private fun clickEvent() {
+        binding.btnFirst.setOnClickListener {
+            binding.ivProfile.setImageResource(R.drawable.ic_back)
+            BackgroundThread().start()
+        }
+        binding.btnSecond.setOnClickListener {
+            binding.ivProfile.setImageResource(R.drawable.ic_back)
+            BackgroundThread2().start()
+        }
+
+        binding.ivProfile.setOnClickListener {
+            BackgroundThread3().start()
+        }
+    }
+
+    private fun getBitmapFromURL(src: String): Bitmap? {
+        return try {
+            val url = URL(src)
+            val connection: HttpURLConnection = url.openConnection() as HttpURLConnection
+            connection.doInput = true  //읽기모드임
+            connection.connect()
+            val input: InputStream = connection.inputStream
+            BitmapFactory.decodeStream(input)
+        } catch (e: IOException) {
+            e.printStackTrace()
+            null
+        }
+    }
+
+    inner class MyHandler : Handler(Looper.getMainLooper()) {
+        override fun handleMessage(msg: Message) {
+            val bundle = msg.data
+            val image = bundle.getString("bitmap")
+            binding.tvCount.text = bundle.getInt("count").toString()
+            image?.let {
+                binding.ivProfile.post {
+                    binding.ivProfile.setImageBitmap(convertBitMap().StringToBitmap(it))
+                }
+            }
```

Comment:
```suggestion
            binding.tvCount.text = bundle.getInt("count").toString()
            val image = bundle.getString("bitmap") ?: return
            binding.ivProfile.post {
                 binding.ivProfile.setImageBitmap(convertBitMap().StringToBitmap(image))
             }
         }
```

수빈이도 마찬가지로 이 부분은 생략해도 좋습니다^^ bundle로 bitmap을 가져오지 못할 경우, return으로 함수를 종료하게된다면 이미지에 대한 null검사 즉, `image?.let{}`이 필요없어지게 됩니다! 개인적으로 let블록이 없어지면서 들여쓰기가 한단계 줄어들기 때문에 가독성이 더 올려준다고 생각합니다!